src/ui/preferences-window/tabs/ControlsTab.swift

```diff
@@ -90,14 +93,28 @@ class ControlsTab {
         return grid
     }
 
-    private static func toShowSection(_ index: Int) -> ([NSView], [NSView], GridView) {
+    private static func shortcutTab(_ index: Int) -> ([NSView], [NSView], GridView) {
+        var holdShortcut = LabelAndControl.makeLabelWithRecorder(NSLocalizedString("Hold", comment: ""), Preferences.indexToName("holdShortcut", index), Preferences.holdShortcut[index], false, labelPosition: .leftWithoutSeparator)
+        holdShortcut.append(LabelAndControl.makeLabel(NSLocalizedString("and press:", comment: "")))
+        let holdAndPress = StackView(holdShortcut)
+        let nextWindowShortcut = LabelAndControl.makeLabelWithRecorder(NSLocalizedString("Select next window", comment: ""), Preferences.indexToName("nextWindowShortcut", index), Preferences.nextWindowShortcut[index], labelPosition: .right)
+        let tab = controlTab(index, [holdAndPress, StackView(nextWindowShortcut)])
+        return (holdShortcut, nextWindowShortcut, tab)
+    }
+
+    //TODO: controls in Gesture tab aren't centered like in Shortcut tab
+    //TODO: 'On release do nothing' doesn't work. See ShortcutStylePreference.focusOnRelease usage and ATShortcut.shouldTrigger usage
```

Comment:
For keyboard shortcut 'On release do nothing' is implemented via skipping shortcut action. I haven't found workaround to the swipe yet. We could just not support such option for swiping.